Tests/CommandsTests/PackageToolTests.swift

```diff
@@ -727,7 +727,7 @@ final class PackageToolTests: XCTestCase {
                 try execute(["config", "get-mirror", "--original-url", "git@github.com:apple/swift-package-manager.git"], packagePath: packageRoot)
             }
 
-            check(stderr: "error: mirror not found\n") {
+            check(stderr: "Error: mirror not found\n") {
```

Comment:
Command-line swiftpm tries to follow compiler like diagnostics so this diff is unexpected.

Reply:
Updated to match the existing diagnostics, thanks!